boards/genericSTM32F334K6.json

```diff
@@ -0,0 +1,36 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F3 -DSTM32F334x8",
+    "f_cpu": "72000000L",
+    "mcu": "stm32f334k6t7",
+    "product_line": "STM32F334X8"
+  },
+  "platforms": "ststm32",
+  "connectivity": [
+    "can"
+  ],
+  "debug": {
+    "jlink_device": "STM32F334K6",
+    "openocd_target": "stm32f3x",
+    "svd_path": "STM32F334x.svd"
+  },
+  "frameworks": [
+    "stm32cube"
```

Comment:
Other frameworks (CMSIS, libopencm3) are not available?

Reply:
Yes probably but as I didn't tested it I didn't want to add it